### ISSUE TITLE
Data loads seem much slower with Net Fetch

### ISSUE THREAD

**Issue by philrz** [brimdata/zui#3105]
## tl;dr

Unfortunately, we failed to notice before the merge of #3069 that those changes there are making data loads much slower For instance, a load that previously took 5 seconds on my Intel Macbook now takes 3+ minutes.

## Details

Repro is with Zui commit cc740ad.

The first video shows how things were performing better at commit 64f50eb right before the changes in #3069 merged. Here I'm doing a Quick Load of the [ZNG-format zed-sample-data](https://github.com/brimdata/zed-sample-data/tree/main/zng). As it shows, the data load takes about 5 seconds.

https://github.com/brimdata/zui/assets/5934157/70e85ba8-db09-4a27-badd-3b106c3a38d6

The second video is at commit cc740ad which is associated with the changes in #3069 that has the app using Net Fetch. As it shows, the same data load is now taking 3+ minutes.

https://github.com/brimdata/zui/assets/5934157/7ae24219-138d-4bbb-a94c-f2009d594985

**Comment by philrz** [comment]
Verified in Zui commit 508a6ae.

https://github.com/brimdata/zui/assets/5934157/075cd86b-1218-43e7-9822-08e7f5f24f0a

As shown in the attached video, the same data load now once again takes under 5 seconds to complete on my Macbook.

Thanks @jameskerr!
